src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowUpdate.cs

```diff
@@ -167,15 +167,21 @@ public override string GetScript()
         /// </summary>
         /// <param name="columnId">Ordinal of the column to revert</param>
         /// <returns>The value that was </returns>
-        public override string RevertCell(int columnId)
+        public override EditRevertCellResult RevertCell(int columnId)
         {
             Validate.IsWithinRange(nameof(columnId), columnId, 0, associatedRow.Count - 1);
 
             // Remove the cell update
+            // NOTE: This is best effort. The only way TryRemove can fail is if it is already
```

Comment:
Eh...... I don't see the benefit of doing that here. It's not repeated anywhere and it's only 2 lines